### PR TITLE
Add Bona fide pusher script for Elixir

### LINKED ISSUE #2513: Pushing bona fide status
Using the python template from #2512, implement a script that reacts to approved events (For a given resource? By the bona fide bot?) and pushes bona fide status to Elixir.

Approved push spec is very simple: we just pass the elixir-id as an URL parameter and that's it:
`https://perun.elixir-czech.cz/rems/?elixirid={elixir_user_id}`

The script should:

- [x] react to `application/approved` events
- [x] look up applicant user id from `event/application`, key `application/applicant`
- [x] POST to `https://perun.elixir-czech.cz/rems/?elixirid={elixir_user_id}` with configurable authentication
- [ ] test with bona-fide-bot, needs a decider with a ResearcherStatus visa
    - [x] developer tests with other profile (google login)
    - [ ] to be tested on Rahti with Mikael, demo also, see next step
        - [x] fix authentication issue first
- [x] set up on the demo instance with mail server configured
    - [x] https://rems-bona-fide-test.rahtiapp.fi is used
    - [x] ask Valtteri for details setting up the container

### PR DESCRIPTION
**Pull request by juholehtonen**
Fixes #2513 

# Checklist for author

Remove items that aren't applicable, check items that are done.

## Reviewability
- [x] Link to issue
- [x] Note if PR is on top of other PR

## Documentation
- [x] Update changelog if necessary
- [x] Update docs/ (if applicable)

## Testing
- [ ] Complex logic is unit tested
- [ ] Valuable features are integration / browser / acceptance tested automatically

## Follow-up
- [x] New tasks are created for pending or remaining tasks
     #2528 is a follow up
     #2836 is a follow up

### CODE REVIEW

File: resources/addons/bona-fide-pusher/config.ini [hunk 1]
```diff
@@ -0,0 +1,7 @@
+; configuration for REMS bona fide pusher
+[default]
+url = localhost
+port = 3008
+elixir_url = https://perun.elixir-czech.cz/rems/
+elixir_userid = rems-external
+elixir_password = 6kBVNYLcLTs8A5XW
```
Comment (Macroz) [inline resources/addons/bona-fide-pusher/config.ini:7]: Should we redact these details from the source or is it OK to publish credentials?

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,9 @@ Changes since v2.24
 ### Breaking changes
 - The columns owneruserid and modifieruserid have been removed as the audit_log effectively serves the same purpose. They were not consistently used. For the attachments the column has been renamed to userid. This is a breaking change for the API as some of these were exposed, but likely not used or useful. (#2823)
 
+### Additions
+- Bona fide pusher script for pushing bona fide status to Elixir.
```
Comment (Macroz) [inline CHANGELOG.md:15]: We could link to the ticket here, i.e., add `(#2513)`.

**Review by Macroz — APPROVED** [review]
Looks good but check if you really want to add the credentials.

File: CHANGELOG.md (hunk 2 above)
Comment (juholehtonen) [reply CHANGELOG.md]: Added.

**Pull request merged**